raiden-ts/src/state.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable @typescript-eslint/no-explicit-any,@typescript-eslint/camelcase */
```

Comment:
Did we completely remove the camelcase requirement or something else changed? could you point to the change?

Reply:
The rule was [completely removed](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/camelcase.md), and replaced by [`@typescript-eslint/naming-convention`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/naming-convention.md), which isn't (yet?) part of `@typescript-eslint/recommended` set of rules we follow, so I had to remove the disables to suppress errors about the unknown rule.